pkg/shopify/config.go

```diff
@@ -0,0 +1,14 @@
+package shopify
+
+import (
+	"fmt"
+)
+
+type Config struct {
+	URL    string
+	APIKey string
+}
+
+func (c *Config) GetIngestrURI() string {
+	return fmt.Sprintf("gorgias://%s?api_key=%s", c.URL, c.APIKey)
```

Comment:
this seems like a copy paste mistake?
```suggestion
	return fmt.Sprintf("shopify://%s?api_key=%s", c.URL, c.APIKey)
```